tests/integration/test_materialize_mysql_database/composes/mysql_5_7_compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '2.3'
+services:
+    mysql5_7:
+        image: mysql:5.7
+        restart: always
+        environment:
+            MYSQL_ROOT_PASSWORD: clickhouse
+        ports:
+          - 33307:3306
+        command: --server_id=100 --log-bin='mysql-bin-1.log' --default-time-zone='+3:00'
```

Comment:
What will happen if ClickHouse and MySQL have different default timezones?

Reply:
For MySQL DateTime type, use ClickHouse default timezone to parse the MySQL data(MySQL time zone)
For the MySQL TimeStamp type, save the seconds of MySQL data to ClickHouse (do nothing)

This is because, for MySQL DateTime type, always sent `YYYY-MM-DD HH: SS: DD` to replica, and for MySQL Timestamp type, always sent 'timestamp seconds' to replica. I think this is reasonable.